### PR TITLE
Add remote platform docs to directv

### PR DESCRIPTION
**Pull request by ctalkington**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/32790
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/directv.markdown [hunk 1]
```diff
@@ -48,3 +49,66 @@ Available services: turn_on, turn_off, media_play, media_pause, media_stop, medi
 | `entity_id`            |      yes | Target a specific media player. Defaults to all.                                                                                                                       |
 | `media_content_id`     |       no | The channel number to change to.                   |
 | `media_content_type`   |       no | A media type. Has to be `channel`.
+
+## Remote
+
+The DirecTV remote platform allows you to send remote control buttons to a DirecTV receiver. It is automatically setup when a DirecTV receiver is configured.
```
Comment (frenck) [inline source/_integrations/directv.markdown]:
```suggestion
The DirecTV remote platform allows you to send remote control buttons to a DirecTV receiver. It is automatically set up when a DirecTV receiver is configured.
```

File: source/_integrations/directv.markdown [hunk 2]
```diff
@@ -48,3 +49,66 @@ Available services: turn_on, turn_off, media_play, media_pause, media_stop, medi
...
+
+At the moment, the following buttons are supported:
+
+- power
```
Comment (frenck) [inline source/_integrations/directv.markdown]: I would suggest making those code blocks, as they are all values that can be used.

i.e.:
```suggestion
- `power`
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @ctalkington!

I've left some comments, could you please take a look? Thanks! 👍

**Comment by frenck** [comment]
Looks good! Awaiting for it to get out of WIP :)

**Comment by frenck** [comment]
@ctalkington What is the status of this docs PR? Since the parent PR has been merged...

**Comment by ctalkington** [comment]
@frenck sorry juggling a bit here between platforms. this is ready to merge

**Comment by frenck** [comment]
Check thanks! 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**